cypress/support/commands.js

```diff
@@ -0,0 +1,35 @@
+Cypress.Commands.add('clickLink', (selector, linkText) => {
+    cy.get(selector).contains(linkText).click();
+})
```

Comment:
Passing selectors directly to Cypress custom commands can be considered as violation of encapsulating logic and readability. 
Consider creating more targeted command.